beacon_node/eth2-libp2p/src/service.rs

```diff
@@ -130,7 +131,7 @@ impl<TSpec: EthSpec> Service<TSpec> {
             }
             SwarmBuilder::new(transport, behaviour, local_peer_id.clone())
                 .peer_connection_limit(MAX_CONNECTIONS_PER_PEER)
-                .executor(Box::new(Executor(tokio::runtime::Handle::current())))
+                .executor(Box::new(Executor(handle)))
```

Comment:
We want to use the task manager here with "libp2p" as the name not just a standard runtime handle